src/main/java/racing/GameApplication.java

```diff
@@ -1,27 +1,24 @@
 package racing;
 
+import java.util.List;
 import racing.domain.Game;
 import racing.domain.MovementStrategy;
-import racing.ui.Message;
+import racing.domain.Participants;
+import racing.domain.RacingGame;
+import racing.domain.RoundCount;
 import racing.ui.InputView;
-import racing.ui.ResultView;
+import racing.ui.Message;
 
 public class GameApplication {
-  public static void main(String[] args){
-    InputView input = new InputView();
-    int carCnt = input.input(Message.INPUT_CAR_COUNT);
-    int tryCnt = input.input(Message.INPUT_TRY_COUNT);
-    MovementStrategy strategy = new MovementStrategy(0, 9);
 
-    Game game = new Game(strategy, carCnt, tryCnt);
-    ResultView result = new ResultView();
-    result.printStart();
+  public static void main(String[] args) {
+    List<String> cars = InputView.inputCar(Message.INPUT_CAR_NAME);
+    int tryCnt = InputView.inputTryCnt(Message.INPUT_TRY_COUNT);
 
-    for (int i = 0; i < tryCnt; i++) {
-      game.play();
-      result.printRoundResult(game.getCars());
-    }
-
-    input.close();
+    MovementStrategy strategy = new MovementStrategy(0, 9);
```

Comment:
```suggestion
    MovementStrategy strategy = new MovementStrategy();
```
빈 생성자 추가했으니 활용해 구현하면 어떨까?